shell/renderer/atom_renderer_client.cc

```diff
@@ -76,14 +76,26 @@ void AtomRendererClient::DidCreateScriptContext(
   // TODO(zcbenz): Do not create Node environment if node integration is not
   // enabled.
 
+  auto* command_line = base::CommandLine::ForCurrentProcess();
+
   // Do not load node if we're aren't a main frame or a devtools extension
   // unless node support has been explicitly enabled for sub frames
-  bool is_main_frame =
-      render_frame->IsMainFrame() && !render_frame->GetWebFrame()->Opener();
+  bool reuse_renderer_processes_enabled =
+      command_line->HasSwitch(switches::kDisableElectronSiteInstanceOverrides);
+  // Considered the window not "opened" if it does not have an Opener, or if a
```

Comment:
I still don't love `is_not_opened`

Reply:
Yeah I'm not a huge fan but I want to group the logic (and this new check goes with the opener check) whilst also not (as per https://github.com/electron/electron/pull/19553#discussion_r309913422) refactoring the entire check because it's a nasty can of worms